README.md

```diff
@@ -74,6 +74,9 @@ The following profiles are currently provided:
 * [`AffinityAndTaints`](#AffinityAndTaints)
 * [`TopologyAndDuplicates`](#TopologyAndDuplicates)
 * [`LifecycleAndUtilization`](#LifecycleAndUtilization)
+  
+Along with the following profiles, which are in Beta and may change:
+* [`BetaLongLifecycle`](#BetaLongLifecycle)
```

Comment:
Why Beta, why not Alpha?

Reply:
Since this is externally visible let's align that with our terminology and use `DevPreview` or `TechPreview`, the former is equivalent of alpha, the latter - beta.